processor/k8sattributesprocessor/metadata.yaml

```diff
@@ -295,6 +311,16 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+    otelsvc_k8s_pod_association:
+      enabled: true
+      description: Number of pod associations
+      stability:
+        level: development
+      unit: "{resources}"
+      attributes: [k8s_telemetry_type, status]
```

Comment:
Should we also consider to add information about which association's field failed or succeeded?

Reply:
agreed, added info about pod, which failed to associate as an attribute